tasks/update-granules-cmr-metadata-file-links/index.js

```diff
@@ -56,6 +58,22 @@ async function updateEachCmrFileAccessURLs(
   }));
 }
 
+async function updateCmrFileInfo(cmrFiles, granulesByGranuleId) {
+  for (const cmrFileObject of cmrFiles) {
```

Comment:
@mikedorfman  - a couple of things - this method is running `await` in a loop, which is probably better re-written to run in parallel, but if intended to run serially requires a linter exception.    

Additionally this method mutates cmrFilesk, which we'd prefer to avoid, generally in favor of writing functions for this sort of thing that are pure/functional.    I'm going to make a quick update to the upstream PR, and will note here when I have that pushed up.

Reply:
Take a look at:  https://github.com/nasa/cumulus/pull/3703/commits/fa329b7f8f4b8aba4a3a3dc1dad1bcbedcd79ef5

Reply:
Ah, I see - that makes sense to me.  @colecu FYI

Reply:
Thanks for the pointers and updates @Jkovarik. My JavaScript is pretty rusty and sometimes bumps up against that "enough to be dangerous" rail. Appreciate the explanations.